Assignment2/Saabertooth1-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,58 @@
+{
+  "@context": {
+      "person": "http://example.org/person",
+      "xsd": "http://www.w3.org/2001/XMLSchema#",
+      "fullName": "http://example.org/fullname",
+      "age":{
+        "@id": "http://example.org/age",
+        "@type":"xsd:integer"
+        },
+      "hometown": {
+        "@id": "http://example.org/hometown",
+        "@type": "@id"
+        },
+      "almaMater": {
+        "@id": "http://example.org/almaMater",
+        "@type": "@id"
+        },
+      "parentsName": {
+        "@id": "http://example.org/parentsName",
+        "@type": "xsd:String"
```

Comment:
typo: String -> string
You may check xsd datatype definitions on the RDF slides.